skr/go.mod

```diff
@@ -5,28 +5,37 @@ go 1.18
 require (
 	github.com/go-logr/logr v1.2.3
 	github.com/kyma-project/kyma-watcher/kcp v0.0.0-20220801122243-5ce523a796aa
+	github.com/kyma-project/manifest-operator/operator v0.0.0-20220722124726-ba2878448c7d
 	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/ginkgo/v2 v2.1.4
 	github.com/onsi/gomega v1.19.0
 	k8s.io/api v0.24.3
+	k8s.io/apiextensions-apiserver v0.24.3
 	k8s.io/apimachinery v0.24.3
 	k8s.io/client-go v0.24.3
+	k8s.io/klog/v2 v2.70.1
```

Comment:
I think you should either use klog, logr, or zap (i prefer zap as it has the easiest integration into middlewares later on), but not multiple at once imo

Reply:
removed klog. I use `go-logr` just to consistent with our implementation elsewhere. `zap` is used by the test suite.